docs/DafnyRef/Types.md

```diff
@@ -2409,7 +2409,7 @@ A one-dimensional array of `n` `T` elements is created as follows:
 a := new T[n];
 ```
 The initial values of the array elements are arbitrary values of type
-`T`. They can also be initialized using a sequence display expression delimited by square brackets, as follows:
+`T`. They can also be initialized using a sequence display expression enclosed in square brackets, as follows:
```

Comment:
I would leave out the 'also', otherwise it's implying the previous sentence already mentioned a way of initializing the values. Also, "a sequence display expression enclosed in square brackets" to me reads like there's an entire sequence display expression in square brackets, even though such as expression already has square brackets.

I think referring to a sequence display expression is a little confusing, since it's not exactly the same as this, so I'd prefer an explicit description like "an ordered list of expressions enclosed in square brackets". If we want to mention "sequence display expression" then I'd like to indicate it's only similar.